app/src/main/java/com/kickstarter/libs/TrackingClient.kt

```diff
@@ -100,7 +101,7 @@ abstract class TrackingClient(@param:ApplicationContext private val context: Con
     }
 
     override fun currentVariants(): JSONArray? {
-        return ConfigUtils.currentVariants(this.config)
+        return this.config?.currentVariants()
```

Comment:
if `config` is null, what does this return? Null?

Reply:
Config cannot be null, it's not optional. In case that happen it will throw NullPointerException()

Reply:
Hmm, then why do we need the safe call operator?

Reply:
what is optional is the return from `currentVariants()` that is a JSONArray?